.changes/unreleased/🎉 Feature-20230116-162527.yaml

```
@@ -1,5 +1,5 @@
 kind: "\U0001F389 Feature"
-body: Add sort, sorted-by flags to role list command.
+body: Add sort flag for represanting data in ascending or descending order and sorted-by flag for specifying by which field to sort data to role list command.
```

```suggestion
body: Add `sort` flag for representing data in ascending or descending order. Add `sorted-by` flag for specifying which field to sort data on with the role list command.
```
Minor spelling and rewording tweak. Thank you!